universe/interface.go

```diff
@@ -194,6 +194,10 @@ type BaseForest interface {
 	// of assets tracked in the base Universe.
 	RootNodes(ctx context.Context) ([]BaseRoot, error)
 
+	RegisterIssuance(ctx context.Context, id Identifier, key BaseKey,
```

Comment:
Doesn't need to be done now, but eventually we'll also want to distinguish from the a multi-verse of only issuance, from one of all transfers for an asset, from one of all transfers for any asset. 

Also notice the other `BaseUniverse` interface doesn't actually take an ID as it's bound to that asset id or group key. As this is a base multi-verse/forst, I think we'd also want to carry that over. You'd then have something similar to the current base uni struct here that is able to fetch the multiverse needed for the action.